e2e-tests/tfm/cypress/e2e/journeys/case/parties/bond-beneficiary.spec.js

```diff
@@ -139,7 +133,8 @@ context('Bond beneficiary URN - User can add, edit, confirm and submit URN to th
         pages.bondBeneficiaryPage.errorSummary().contains('Enter a minimum of 3 numbers');
         pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
         pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
-
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(60000);
```

Comment:
We should be able to remove all the `cy.wait` lines now that the data team have increased the rate limit